src/components/ui/copy/copy.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styled, { css } from 'styled-components';
+import { IconButton } from '../buttons/icon-button';
+import { Icon } from '../icon/icon';
+
+const Default_Clicked_Duration = 3000;
```

Comment:
Constants always upper-case: `DEFAULT_CLICKED_DURATION`.

Also not 100% happy with the name. It doesn't clearly indicate what it does and the unit is missing. 
Better could be: `SHOW_SUCCESS_NOTIFICATION_MS`

Reply:
Ohs 🧹